src/menu/tooltipmenu.js

```diff
@@ -134,7 +134,7 @@ function topCenterOfSelection() {
 }
 
 function bottomCenterOfSelection() {
-  let range = window.getSelection().getRangeAt(0), rects = range.getClientRects()
+  let range = this.pm.root.getSelection().getRangeAt(0), rects = range.getClientRects()
```

Comment:
Where does `this` come from? Shouldn't it follow the same signature as `topCenterOfSelection`?